### ISSUE TITLE
Custom popup positions 

### ISSUE THREAD

**Issue by simopaasisalo** [Leaflet/Leaflet.markercluster#722]
Hi,

I'm showing statistical data about clusters on mouse hover in an `L.PopUp`. The icon is a custom `L.DivIcon`, the popup is created by `cluster.bindPopup('popupcontent)` in the `iconCreateFunction` and handled with `clustermouseover/-out`. 

I would like for the popup to open on top of the icon, so as not to cover the it. I've tried setting a `popUpAnchor` to the DivIcon, but still the popups open at the middle of the clustered region.

As far as I can tell, the ClusterGroup doesn't have an option to set popUp offset/anchor position. Are there any alternatives I could try? The only possibility I see is to maybe bind and open the popup on the mouse event with the custom lat-lng, but maybe there is a simpler option?

TL;DR: PopUps always open in the middle of the region, ignoring DivIcon popUpAnchor

**Comment by danzel** [comment]
This sounds more like a leaflet bug?
Make a repro case that doesn't involve marker cluster and post it on leaflet.

**Comment by ghybs** [comment]
Hi,

Just in case, make sure you do not have a typo / case issue: option is [`popupAnchor`](http://leafletjs.com/reference.html#divicon-popupanchor) (note the lowercase `u`)

**Comment by simopaasisalo** [comment]
@ghybs Good point, but wasn't the case. 

I'll see about reproducing this and posting on Leaflet

**Comment by Golmote** [comment]
Sorry for digging out this issue, I happen to have run into the same situation today.

@danzel I feel like the issue lies somewhere in the MarkerCluster prototype, where it "pretends to be an icon".

https://github.com/Leaflet/Leaflet.markercluster/blob/93bf5408ad84b74ea42f55ccb6ca0450daa41427/src/MarkerCluster.js#L92-L102

I believe it would be possible to implement `_getPopupAnchor` on MarkerCluster to fix this issue.
